boards/common/arduino-zero/Makefile.features

```diff
@@ -0,0 +1,17 @@
+CPU = samd21
+CPU_MODEL = samd21g18a
```

Comment:
```suggestion
CPU_MODEL ?= samd21g18a
```

Just in case someone builds a clone with a slightly different model

Reply:
Done